RELEASE_NOTES.md

```diff
@@ -1,7 +1,8 @@
 # Release notes
 
-## New in git-machete 3.25.3
+## New in git-machete 3.26.0
 
+- added: better detection of squash merges and rebases, controlled by argument `--squash-merge-detection={none,simple,exact}` and git config key `machete.squashMergeDetection` (contributed by @gjulianm)
```

Comment:
New flag added, bumping minor version (as per [versioning policy](https://github.com/VirtusLab/git-machete/blob/master/CONTRIBUTING.md#versioning)).

Reply:
Also, mention the deprecation of `--no-detect-squash-merges` + which commands are affected by the flag changes